lib/src/worker_protocol.pb.dart

```diff
@@ -13,12 +13,13 @@ class Input extends GeneratedMessage {
   static final BuilderInfo _i = new BuilderInfo('Input')
```

Comment:
Do we really want to be reformatting this file?  It's generated code, so next time we regen it we'll un-do the formatting.